app/ui.js

```diff
@@ -949,6 +949,7 @@ const UI = {
     // further information at https://developer.mozilla.org/en-US/docs/Web/API/Clipboard
 
     writeLocalClipboard(text) {
+        Log.Debug(">> UI.writeLocalClipboard: " + text.substr(0, 40) + "...");
```

Comment:
do we need logs?

Reply:
Yes. It's noVNC logs. It will be shown in console only if user set noVNC logs settings to "Debug".